sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -159,15 +159,33 @@ public DefaultAndroidEventProcessor(
     if (event.getContexts().getDevice() == null) {
       event.getContexts().setDevice(getDevice());
     }
-    if (event.getContexts().getOperatingSystem() == null) {
-      event.getContexts().setOperatingSystem(getOperatingSystem());
-    }
+
+    mergeOS(event);
 
     setSideLoadedInfo(event);
 
     return event;
   }
 
+  private void mergeOS(final @NotNull SentryEvent event) {
+    final OperatingSystem currentOS = event.getContexts().getOperatingSystem();
+    final OperatingSystem androidOS = getOperatingSystem();
+
+    // make Android OS the main OS using the 'os' key
+    event.getContexts().setOperatingSystem(androidOS);
+
+    if (currentOS != null) {
```

Comment:
Is this here accidently?
The changelog has only the native bump

Reply:
no, its a change due to a breaking change on sentry-native, see `Motivation and Context`: just updated it.